docker/vernemq/Dockerfile

```diff
@@ -47,7 +47,7 @@ EXPOSE 1883 8883 8080 44053 4369 8888 \
 
 VOLUME ["/vernemq/log", "/vernemq/data", "/vernemq/etc"]
 
-HEALTHCHECK CMD vernemq ping | grep -q pong
+HEALTHCHECK CMD wget --no-verbose --tries=1 --spider http://localhost:8888/health || exit 1
```

Comment:
Can we use `curl` here instead of `wget`?

Reply:
Alternative with `curl` could be:
```HEALTHCHECK CMD  [[ "$(curl -o /dev/null -s -w "%{http_code}\n" http://localhost:8888/health)" == "200" ]]```

The command’s exit status indicates the health status of the container. The possible values are:
0: success - the container is healthy and ready for use
1: unhealthy - the container is not working correctly

That's why this `curl` compare if return http code is `"200"`

Reply:
I prefer `curl` command. It looks more complicated on a first look, but it explains better what it does.

But I would prefer even more following @jpetazzo's [instructions](https://github.com/jpetazzo/container.training/blob/master/slides/swarm/healthchecks.md#health-checks-and-auto-rollbacks) with `curl -f`.

Reply:
changed